src/web/components/SignInGate/displayRule.ts

```diff
@@ -72,3 +74,20 @@ export const isValidTag = (CAPI: CAPIBrowserType): boolean => {
 		CAPI.tags.map((tag) => tag.id).includes(invalidTag),
 	);
 };
+
+export const canShowCopyOpt = (
```

Comment:
is this the same as the main-variant `canShow`?
If so we should export and use that one

Reply:
It's slightly different as the main variant `canShow` does not pass in a count for the `hasUserDismissedGate` function. It could be that it's an error on the main variant.

Reply:
Ah yeah, thats a mistake on the main variant. It should be the same as this, so if they've dismissed more than 5 times we stop showing the gate

Reply:
I have refactored the code so that the `canShow` function is only defined once